src/main/java/seedu/address/model/RoomBook.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.room.Room;
+import seedu.address.model.room.UniqueRoomList;
+
+public class RoomBook implements ReadOnlyRoomBook {
+
+    private final UniqueRoomList rooms;
+
+    {
+        rooms = new UniqueRoomList();
+    }
+
+    public RoomBook() {}
+
+    /// list level operations
+    public ObservableList<Room> getAvailableRooms(ObservableList<Room> unavailableRooms) {
```

Comment:
This is for us to get available rooms based on which Rooms we currently have that are booked. But the booking class only contains the room ids and not the Room object so the input may be a list of integer IDs instead? Depends on how we implement the rest so I think can leave it for now!